### PR TITLE
[12.0] [MIG] maintenance_equipment_hierarchy

### PR DESCRIPTION
**Pull request by NuriaXifre**
Migration of the module maintenance_equipment_hierarchy to v12
cc ~ @ForgeFlow

### CODE REVIEW

**Review by etobella — APPROVED** [review]
There is a new commit on 11.0, cherry pick it please :smile: 
No major changes

**Comment by etobella** [comment]
@NuriaMForgeFlow Something is wrong, probably because parent_path is missing. Can you fix it please?

**Comment by etobella** [comment]
Just two small lint issues and all will be green:
`@api.one` is deprecated and version should be `12.0.1.0.0`
Can you check it please and all will be fine for the merge :smile:

**Comment by etobella** [comment]
I think you took also `maintenance_team_hierarchy`

**Comment by etobella** [comment]
@NuriaMForgeFlow if you squash the last two commits I think everything will be fine and ready :smile:

File: maintenance_equipment_hierarchy/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     'author': 'Eficent, Odoo Community Association (OCA)',
     'website': 'http://github.com/OCA/maintenance',
     'category': 'Equipments, Assets, Internal Hardware, Allocation Tracking',
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.1.0',
```
Comment (LoisRForgeFlow) [inline maintenance_equipment_hierarchy/__manifest__.py]: 12.0.1.0.0

**Review by LoisRForgeFlow — COMMENTED** [review]
add yourself as a contributor

File: maintenance_equipment_hierarchy/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Maintenance Equipments Hierarchy',
+    'summary': 'Manage equipment hierarchy',
+    'author': 'Eficent, Odoo Community Association (OCA)',
```
Comment (LoisRForgeFlow) [inline maintenance_equipment_hierarchy/__manifest__.py]: ForgeFlow here and in corpyrights

File: maintenance_equipment_hierarchy/__manifest__.py (hunk 1 above)
Comment (etobella) [reply maintenance_equipment_hierarchy/__manifest__.py]: This is missing @NuriaMForgeFlow

**Review by jarroyomorales — APPROVED** [review]
LGTM!

**Comment by dalonsod** [comment]
@AdriaGForgeFlow @JordiBForgeFlow I'm interested in v13 migration for this addon, but I have a doubt: the current license for this module is LGPL, instead of AGLP, which is the default license for this repo. Is that correct, or a mistake?
Anyway, which license should be filled fot this addon in v13?

Thanks in advance!

**Comment by AdriaGForgeFlow** [comment]
> Anyway, which license should be filled fot this addon in v13?

The license is correct. You should keep it as it is (LGPL).
Let me know when the migration is ready for review or if you have any issue.

**Comment by dalonsod** [comment]
Thanks, @AdriaGForgeFlow

**Pull request merged**